argilla-server/src/argilla_server/models/mixins.py

```diff
@@ -185,9 +185,9 @@ async def delete(self, db: AsyncSession, autocommit: bool = True) -> Self:
 
     @classmethod
     async def delete_many(
-        cls, db: AsyncSession, params: List["BinaryExpression"], autocommit: bool = True
+        cls, db: AsyncSession, where: List["BinaryExpression"], autocommit: bool = True
```

Comment:
We're using `conditions` in other functions:

```suggestion
        cls, db: AsyncSession, conditions: List["BinaryExpression"], autocommit: bool = True
```